agent/agent_test.go

```diff
@@ -928,6 +1138,20 @@ func (s *AgentSuite) getPanicLogs() []string {
 	return panicLogs
 }
 
+// getIdleTimeoutLogs returns logs setting the idle timeout.
+func (s *AgentSuite) getIdleTimeoutLogs() []string {
```

Comment:
This can be replaced by just using `checkMockLogs` to look for the strings `Set idle timeout for ...` instead of making its own function.